source/mainnet/tutorials/using-ID-in-dApps/zk-proofs-tutorial.rst

```diff
@@ -0,0 +1,313 @@
+.. _zk-proofs-tutorial:
+
+===================
+ZK proofs structure
```

Comment:
The goal of the tutorial is to teach the reader more about zk proof generation (not only about the structure of zk proofs), so maybe we can reflect that in the header.
```suggestion
ZK proof generation
```